lib/happy_gemfile.rb

```diff
@@ -82,9 +82,9 @@ def self.organize_groups lines=nil
       organized << "\n"
 
       (groups.keys - [:general, :not_gems]).each do |group|
-        organized << "group #{group_line(group)} do" << "\n"
+        organized << "\n" << "group #{group_line(group)} do" << "\n"
```

Comment:
Wouldn't that create a redundant empty line before the first group?

Reply:
Correct. It is fixed now

Reply:
I invoked `happy_gemfile all` on the Gemfile in this repository and there are two empty lines at the end of the file. I suppose we need to rethink how to add the newline characters.